packages/block-library/src/post-content/index.php

```diff
@@ -46,10 +46,33 @@ function render_block_core_post_content( $attributes, $content, $block ) {
 		$content .= wp_link_pages( array( 'echo' => 0 ) );
 	}
 
+	$ignored_hooked_blocks = get_post_meta( $post_id, '_wp_ignored_hooked_blocks', true );
```

Comment:
The trick for the first and last child of the Post Content block works like a charm!